### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by davidkpiano**
Fix links

### CODE REVIEW

**Review by codepossible — COMMENTED** [review]
Thanks for the fix. Can you make these relative paths, instead of absolute?

**Review by codepossible — APPROVED** [review]

**Comment by codepossible** [comment]
Added more reviewers...

File: data-generator/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Data Generator
 
-This is a data generator to create and submit [on hand](../assets/input-onhand-schema.json) and [shipment](../assets/input-shipment-schema.json) documents to Cosmos DB.
+This is a data generator to create and submit [on hand](https://github.com/cse-southeast-devcrew/openhack-durable-entities/blob/master/docs/json/samples/input-onhand-sample.json) and [shipment](https://github.com/cse-southeast-devcrew/openhack-durable-entities/blob/master/docs/json/samples/input-shipment-sample.json) documents to Cosmos DB.
```
Comment (davidkpiano) [inline data-generator/README.md]:
```suggestion
This is a data generator to create and submit [on hand](docs/json/samples/input-onhand-sample.json) and [shipment](docs/json/samples/input-shipment-sample.json) documents to Cosmos DB.
```

File: data-generator/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Data Generator
 
-This is a data generator to create and submit [on hand](../assets/input-onhand-schema.json) and [shipment](../assets/input-shipment-schema.json) documents to Cosmos DB.
+This is a data generator to create and submit [on hand](docs/json/samples/input-onhand-sample.json) and [shipment](docs/json/samples/input-shipment-sample.json) documents to Cosmos DB.
```
Comment (davidkpiano) [inline data-generator/README.md]:
```suggestion
This is a data generator to create and submit [on hand](../docs/json/samples/input-onhand-sample.json) and [shipment](../docs/json/samples/input-shipment-sample.json) documents to Cosmos DB.
```

**Review by brandonh-msft — APPROVED** [review]

**Pull request merged**